### PR TITLE
Components

### PR DESCRIPTION
**Pull request by bnazmi**

### CODE REVIEW

File: src/client/src/Components/Input/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component, Fragment } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './style.css';
+
+const Input = (props) => {
+    
+    return (
+        <div className="input-container">     
+            <FontAwesomeIcon icon={props.icon} />
+            <input placeholder={props.placeholder} name={props.name} value={props.value} type={props.type} />
```
Comment (Eslam3bed) [inline src/client/src/Components/Input/index.js]: you could do something like the spread syntax
```javascript 
<input {...props} />
```

File: src/client/src/App.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+       <Fragment>
+        {/* <SignIn /> */}
```
Comment (Eslam3bed) [inline src/client/src/App.js]: no commenting code on the master branch

File: src/client/src/Components/Input/index.js (hunk 1 above)
Comment (bnazmi) [reply src/client/src/Components/Input/index.js]: it's great, this first time I used it!

File: src/client/src/App.js (hunk 2 above)
Comment (bnazmi) [reply src/client/src/App.js]: okay

**Pull request merged**